Crashlytics/Shared/FIRCLSNetworking/FIRCLSFABNetworkClient.m

```diff
@@ -209,20 +211,20 @@ - (void)startDownloadTaskWithRequest:(NSURLRequest *)request
                                       }
 
                                       [self
-                                        runAfterRetryValueFromResponse:response
-                                                              attempts:tries
-                                                                 block:^{
-                                                                   [self
-                                                                       startDownloadTaskWithRequest:
-                                                                           request
-                                                                                         retryLimit:
-                                                                                             retryLimit
-                                                                                              tries:
-                                                                                                  (tries +
-                                                                                                   1)
-                                                                                  completionHandler:
-                                                                                      completionHandler];
-                                                                 }];
+                                          runAfterRetryValueFromResponse:response
+                                                                attempts:tries
+                                                                   block:^{
+                                                                     [self
+                                                                         startDownloadTaskWithRequest:
+                                                                             request
+                                                                                           retryLimit:
+                                                                                               retryLimit
+                                                                                                tries:
+                                                                                                    (tries +
+                                                                                                     1)
+                                                                                    completionHandler:
+                                                                                        completionHandler];
+                                                                   }];
```

Comment:
It doesn't get prettier with clang-format updates 😄

Reply:
Every update we stray further from the left margin 😨